client/src/AppRoot.tsx

```diff
@@ -36,49 +37,72 @@ const AppRoot = () => {
     history.push(routerPaths.root)
   }
 
-  const addElection = (newElection: MailElection) => {
-    const newElections = (mailElections || []).concat([newElection])
-    setMailElections(newElections)
-  }
+  const updateMailElections = useCallback(async () => {
```

Comment:
`update` in the name is a tad confusing, maybe `load` ?

Reply:
"update"d! 😛